css/css-flexbox-exercise-2.css

```diff
@@ -0,0 +1,104 @@
+body {
+    margin: 0% 5%;
+    
+}
+
+body {
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: #5789DE;
+}
+
+
+.body-app {
+    margin-top:20px;
+    max-width: 600px;
+    height: 800px;
+    border-top: 10px solid #1E1E1E;
+    border-right: 10px solid #1E1E1E;
+    border-bottom: 80px solid #1E1E1E;
+    border-left: 10px solid #1E1E1E;
+}
+
+.header-top {
+    height: 100px;
+    background-color: #E7D05C;
+    display:flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+h1 {
+    font-size: 50px;
+    font-weight: 500;
+}
+
+.ui_button  {
+
+    color:#fff;
+    background-color: rgba(0, 0, 0, 0.253);
+    border: 0px;
+    margin: 10px;
+    padding: 5px 10px;
+
+}
+
+.ui_button:hover {
+    color: rgba(0, 0, 0, 0.253);
+    background-color: #fff;
+}
+
+
+.text {
+
+    background-color: #fff;
+    height: 550px;
+    overflow: auto;
+}
+
+.text p {
+
+    margin: 0px 15px;
+    padding-top: 10px;
+}
+
+
+.ui__navigation {
+
+    display: flex;
+    height: 150px;
+    justify-content: space-around;
+    background-color: #E7D05C;
+    align-items: center;
+   
+
+}
+
+.nav_item {
+   
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: 5px;
+    border: 1px solid rgba(0, 0, 0, 0.253);
+    width:100%;
+    height:100%;
+    color:#fff;
+
+}
+
+.nav_item:hover {
+
+    
+    border-bottom: 10px solid rgba(0, 0, 0, 0.253);
+    border-top: 0px;
+    
+}
+
+.nav_item p {
+    color:#000;
+    font-size: 20px;
+}
```

Comment:
tutaj naprawde fajnie, oprócz takich pierdół, np. styluejsz jeszcze po tagach, np `p` to ogólnie git, fajnie ze robisz juz takie ogólne klasy  do poszczególnych elementów aplikacji jak `button` i `nav_item` a nie tworzysz klasy per button i każda z nich ma praktycznie te same style, bardzo na plus :) ew jeszcze formatowanie, nie wiem czy to github tak wyświetla, ale np `.nav__item:hover` ma jakieś 3 puste linie na poczatku selektora albo `.ui__navigation` 3 na koncu

Reply:
Pewnie coś stamtąd usuwałem, ale spacji już nie usunąłem. U mnie w oryginalnym pliku jest tak samo. Będę zwracał na to uwagę